contributors/devel/api-conventions.md

```diff
@@ -563,6 +566,43 @@ Strategic Merge Patch also supports special operations as listed below.
 
 ### List Operations
 
+#### List of Scalars
+
+We have two merge strategies for list of scalars: replace and merge. Default strategy is replace,
+which will preserve the order; while merge strategy works like an unordered set. We call a list
+using merge strategy a merging list.
+
+##### Merging List of Scalars
+
+To add items "a" and "b" in the finalizers list which is a merging list of strings, the patch is:
+
+```yaml
+finalizers:
```

Comment:
They are in the tags of go struct like [this](https://github.com/kubernetes/kubernetes/blob/v1.6.0-alpha.0/pkg/api/v1/types.go#L202). It should have `patchStrategy:"merge"`